src/content-helper/dashboard-widget/top-posts/component-list.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { Spinner } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import DashboardWidgetProvider from '../provider';
+import TopPostListItem from './component-list-item';
+import { TopPostData } from './model';
+import { ContentHelperError } from '../../../blocks/content-helper/content-helper-error';
+import { getDateInUserLang, SHORT_DATE_FORMAT } from '../../../blocks/shared/utils/date';
+
+const FETCH_RETRIES = 3;
```

Comment:
Using this in multiple places .... maybe we should define this in some common location 🤔

Reply:
Related to [this](https://github.com/Parsely/wp-parsely/pull/1305#discussion_r1086950059). I'll be noting this for that PR.